packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+import { authenticationProviders } from '../../stores/authenticationProviders';
+import { faCircle, faRightFromBracket } from '@fortawesome/free-solid-svg-icons';
+import SettingsPage from './SettingsPage.svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+</script>
+
+<SettingsPage title="Authentication">
+  <div class="container mx-auto">
+    <!-- Authentication Providers table start -->
+    <div class="w-full">
+      {#each $authenticationProviders as provider}
+        <!-- Registered Authentication Provider row start -->
+        <div class="flex flex-col w-full">
+          <div class="flex rounded-lg border-0" style="background-color: rgb(39 39 42 / var(--tw-bg-opacity))">
+            <!-- Authentication Provider name and status item start -->
+            <div class="pl-10 pt-2 pb-2 text-sm w-1/3 m-auto">
+              <div class="flex flex-col">
+                <div class="flex items-center text-xl w-full h-full">
+                  {provider.displayName}
+                </div>
+                <div class="flex flex-row items-center w-full h-full">
+                  <dif>
+                    <Fa
+                      class="h-3 w-3 text-md mr-2 text-{provider.accounts.length > 0 ? 'green' : 'gray'}-500"
+                      icon="{faCircle}" />
+                  </dif>
+                  <div class="uppercase text-xs text-{provider.accounts.length > 0 ? 'green' : 'gray'}-500">
+                    <span>
+                      {provider.accounts.length > 0 ? 'Logged in' : 'Logged out'}
+                    </span>
+                  </div>
+                  {#if provider.accounts.length > 0}
+                    <a
+                      class="pl-2 hover:cursor-pointer hover:text-white text-white"
+                      href="javascript:undefined"
+                      on:click="{() =>
+                        window.requestAuthenticationProviderSignOut(provider.id, provider.accounts[0].id)}">
```

Comment:
we should use `<button>` for all `on:click` calls so it's compliant with a11y

(it avoids also href="javascript:undefined")